apps/etterlatte-klage/build.gradle.kts

```diff
@@ -3,13 +3,18 @@ plugins {
     id("etterlatte.rapids-and-rivers-ktor2")
 }
 
+val klageKodeverkVersion = "1.6.10"
```

Comment:
Lurer på om eg helst ser denne i toml-fila. Ser vel ikkje heilt for meg at vi vil ha denne andre stadar enn her, men fint å vera heilt konsekvent.